add-permissions.js

```diff
@@ -66,7 +66,7 @@ module.exports = class AwsAddLambdaAccountPermissions {
         }
 
         if (role) {
-          [].concat(role).forEach(({ name, principals }) => {
+          [].concat(role).forEach(({ allowTagSession, maxSessionDuration, name, principals }) => {
```

Comment:
```suggestion
          [].concat(role).forEach(({ allowTagSession, maxSessionDuration = 3600, name, principals }) => {
```

Reply:
Does that work in node 8.x?

https://github.com/rschick/serverless-plugin-lambda-account-access/blob/9e15aa34f2987d59d7324423bdf8bc8c963efc1e/package.json#L5

Reply:
It does!!